### DISCUSSION TITLE
Audio generator stereo signals

### DISCUSSION THREAD

**Discussion by franjf97** [pschatzmann/arduino-audio-tools#848] [Q&A] [answered]
Hi! I managed to run a Noise Generator as output in an I2S stereo module in an ESP32 and it works as expected. But I don't know if the noise is generated separately for each channel or if it's the same samples for both. In my application I need the same exact signal in both channels, since I'm using the stereo output as an "splitter" of the same noise. Is there a way a can configure that? Thanks!

**Comment by pschatzmann** [comment, marked as answer]
Why don't you just make a test and output the signal as CSV? Then you will see if they are the same or different.
If you set the channels to 1 and you will always get the same output on both i2s output channels
